web/src/components/Selector.tsx

```diff
@@ -37,12 +37,13 @@ const Element = styled(Button)<{ $selected: boolean; $enabled: boolean }>`
   ${props =>
     props.$selected
       ? 'font-weight: 700;'
-      : `:hover {
+      : `&:hover {
           font-weight: 700;
           border-radius: 0;
         }`}
 `
 
+// TODO: ask in the PR if anybody knows what this is for
 const BoldSpacer = styled.div`
```

Comment:
Does anybody know what this is for? I couldn't find a difference when I removed it.